hermes/file_gen_test.go

```diff
@@ -0,0 +1,39 @@
+package hermes
+
+import(
+	//"fmt"
+	"testing"
+	"math/rand"
+	"strconv"
+	"time"
+	//"crypto/sha1"
+)
+
+
+func TestChecksum(t *testing.T){
+	var global_contents string = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+	var result_expected string = "_68f3caf439065824dcf75651c202e9f7c28ebf07"
+	file := HermesFile{}
+	file.contents = global_contents;
```

Comment:
If you're just using global_contents to assign your test contents to file.contents, then consider just skipping the declaration of global_contents and just assigning the string to file.contents directly. 

(P.S. Remember to delete line 10 if/when accepting the suggested change below)

```suggestion
	file.contents =  "jhfvjhdfjhfjjhjhdfvjvcvfjh"
```